src/maps/london/starter.ts

```diff
@@ -0,0 +1,29 @@
+import { GameStarter } from "../../engine/game/starter";
+import { assert } from "../../utils/validate";
+
+export class LondonStarter extends GameStarter {
+  protected onStartGame(): void {
+    super.onStartGame();
+
+    const bag = [...this.bag()];
```

Comment:
I'm not sure what rule is being accessed here, but can't you just address this with numStartingCubes?

Reply:
From what I could tell from numStartingCubes is only respected for cities.

Reply:
Oh, I see... Maybe you can override `drawCubesFor`? Then if it's a land space and the name matches, it gets a cube.

Reply:
Oh yea, that sounds cleaner. I'll do that revision later tonight.